### PR TITLE
Added US38, US39 and their tests, along with folders for their seeds

### PR DESCRIPTION
**Pull request by pdamianov-dev**

### CODE REVIEW

**Review by samanthainneo99 — APPROVED** [review]
Looks good to me, but I think you need to add to the test suite

File: src/us38.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+def upcomingDate (dt):
+    
+    if dt == '':
+        return False
+    
+    today = datetime.today()
+    upcomingDays = today + timedelta(days = 30)
+    
+    dt = datetime.strptime(dt + " " + date.today().strftime('%Y'), '%d %b %Y')
```
Comment (ming1in) [inline src/us38.py:27]: Just pushed up a change to Gedcom that take care of parsing datetimes, so don't need to do this anymore

File: src/us39.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+# In[60]:
+
+
+def us39 (gedcom_file):
+    
+    gedcom = Gedcom(gedcom_file)
+    
+    upcomingAnn = [family.uid for family in gedcom.families 
+                     if (family.marriage != '' and upcomingDate(datetime.strftime(datetime.strptime(family.marriage, "%d %b %Y"), "%d %b")))]
```
Comment (ming1in) [inline src/us39.py:24]: same here with the `datetime.strptime()`

**Review by ming1in — APPROVED** [review]
make sure ya add your test to `test_suite.py` and Gedcom handles parsing datetimes now, so don't need to do that anymore. Overall, everything else looks good, but address changes before merge.

**Review by samanthainneo99 — APPROVED** [review]
Looks good:)

**Pull request merged**